internal/pkg/plugin/trellogithub/create.go

```diff
@@ -11,24 +11,19 @@ func Create(options map[string]interface{}) (map[string]interface{}, error) {
 		return nil, err
 	}
 
-	api := tg.GetApi()
-	log.Infof("API is: %s.", api.Name)
-	ws := defaultWorkflows.GetWorkflowByNameVersionTypeString(api.Name)
-
-	for _, w := range ws {
-		if err := tg.renderTemplate(w); err != nil {
-			return nil, err
-		}
-		if err := tg.client.AddWorkflow(w, tg.options.Branch); err != nil {
-			return nil, err
-		}
+	if err := tg.client.AddWorkflow(trelloWorkflow, tg.options.Branch); err != nil {
+		return nil, err
 	}
+
 	log.Success("Adding workflow file succeeded.")
-	trelloIds, err := tg.CreateTrelloItems()
-	if err != nil {
-		return nil, err
+
+	trelloIds := &TrelloItemId{
```

Comment:
The instance with struct `TrelloItemId` should be `trelloItemId` or something else but not `xxxs`, if the plural is used, it seems like the instance with `[]TrelloItemId` or `map[xxx]TrelloItemId`